### PR TITLE
Remove excessive permission checks for get service

### PR DESCRIPTION
**Pull request by meng-han**
This excessive permission check may introduce unnecessary warnings.

### CODE REVIEW

File: confidant/routes/services.py [hunk 1]
```diff
@@ -209,16 +209,18 @@ def get_service(id):
...
-        action='get',
-        resource_id=id,
-    )
+    if action == 'metadata':
+        permissions['metadata'] = acl_module_check(
+            resource_type='service',
+            action='metadata',
+            resource_id=id,
+        )
+    elif action == 'get':
```
Comment (ruwaifaa) [inline confidant/routes/services.py:218]:
```suggestion
    else:
```

Comment (meng-han) [reply confidant/routes/services.py:218]: there's a 'update' in the permission map, so i keep it more strict

**Review by ruwaifaa — APPROVED** [review]

File: confidant/routes/services.py (hunk 1 above)
Comment (meng-han) [reply confidant/routes/services.py:218]:
```
  permissions = {
        'metadata': False,
        'get': False,
        'update': False,
    }
```

Comment (ruwaifaa) [reply confidant/routes/services.py:218]: action can't take any other values besides "get" and "metadata" tho. np. just a nit tho

**Pull request merged**